core/src/plugins/filed/python/ldap/bareos-fd-ldap.py

```diff
@@ -384,10 +384,13 @@ def get_next_file_to_backup(self, savepkt):
                 # if there is nothing return an error.
                 try:
                     res_type, res_data, res_msgid, res_controls = next(self.resultset)
-                    self.ldap_entries = collections.deque(res_data)
-                except ldap.NO_SUCH_OBJECT:
+                    if res_type == ldap.RES_SEARCH_ENTRY:
+                        self.ldap_entries = collections.deque(res_data)
+                except ldap.NO_SUCH_OBJECT as e:
+                    _log_ldap_err(e, bareosfd.M_FATAL, "error searching base DN")
                     return bareosfd.bRC_Error
                 except StopIteration:
+                    bareosfd.DebugMessage(1, "premature end of LDAP result set\n")
```

Comment:
I would use a job message here because otherwise backups will show up as `Ok -- with warnings` and we wont know why.

Reply:
Good point, changed it accordingly.